mevislab.github.io/content/tutorials/dataobjects/markers/markerexample1.md

```diff
@@ -26,14 +26,14 @@ We changed the names of the modules `SoView2DMarkerEditor` and `XMarkerLIstConta
 
 As a next step, add two more modules: `SoView2DMarkerEditor` and `XMarkerLIstContainer`.
 
-Change their names and the marker color to *green* and connect them as shown. We also like to change the mouse button you need to press, to create a marker. This enables to place both types of markers, the red ones and the green ones. In order to do this, open the panel of `GreenMarkerList`. Under *Buttons* you can adjust, which button needs to be pressed in order to place a marker. Select here the *Button2* (the middle button of your mouse {{< mousebutton "middle" >}}) instead of *Button1* (the left mouse button {{< mousebutton "left" >}}).
+Change their names and the marker color to *green* and connect them as shown. We also like to change the mouse button you need to press, to create a marker. This enables to place both types of markers, the red ones and the green ones. In order to do this, open the panel of `GreenMarker`. Under *Buttons* you can adjust, which button needs to be pressed in order to place a marker. Select here the *Button2* (the middle button of your mouse {{< mousebutton "middle" >}}) instead of *Button1* (the left mouse button {{< mousebutton "left" >}}).
```

Comment:
to create a marker -> in order to create a marker
This enables -> This allows (or: enables the feature)
Select here -> Select